apps/server/src/modules/news/uc/news.uc.spec.ts

```diff
@@ -106,9 +106,9 @@ describe('NewsUc', () => {
 	describe('findAllByUser', () => {
```

Comment:
Where is the case for findAllUnpublished ?

Reply:
Added tests